src/status_im2/common/home/actions/view.cljs

```diff
@@ -54,294 +54,323 @@
+          :danger?             false
+          :accessibility-label :fetch-messages
+          :sub-label           nil
+          :chevron?            true}))
 
 ;; TODO(OmarBasem): Requires design input.
 (defn pinned-messages-entry
   []
-  (entry {:icon      :i/pin
-          :label     (i18n/label :t/pinned-messages)
-          :on-press  #(js/alert "TODO: to be implemented, requires design input")
-          :danger?   false
-          :sub-label nil
-          :chevron?  true}))
+  (entry {:icon                :i/pin
+          :label               (i18n/label :t/pinned-messages)
+          :on-press            #(js/alert "TODO: to be implemented, requires design input")
+          :danger?             false
+          :accessibility-label :pinned-messages
+          :sub-label           nil
+          :chevron?            true}))
 
 (defn remove-from-contacts-entry
   [contact]
-  (entry {:icon      :i/remove-user
-          :label     (i18n/label :t/remove-from-contacts)
-          :on-press  #(hide-sheet-and-dispatch [:contact.ui/remove-contact-pressed contact])
-          :danger?   false
-          :sub-label nil
-          :chevron?  false}))
+  (entry {:icon                :i/remove-user
+          :label               (i18n/label :t/remove-from-contacts)
+          :on-press            #(hide-sheet-and-dispatch [:contact.ui/remove-contact-pressed contact])
+          :danger?             false
+          :accessibility-label :remove-from-contacts
+          :sub-label           nil
+          :chevron?            false}))
 
 ;; TODO(OmarBasem): Requires design input.
 (defn rename-entry
   []
-  (entry {:icon      :i/edit
-          :label     (i18n/label :t/rename)
-          :on-press  #(js/alert "TODO: to be implemented, requires design input")
-          :danger?   false
-          :sub-label nil
-          :chevron?  false}))
+  (entry {:icon                :i/edit
+          :label               (i18n/label :t/rename)
+          :on-press            #(js/alert "TODO: to be implemented, requires design input")
+          :danger?             false
+          :accessibility-label :rename-contact
+          :sub-label           nil
+          :chevron?            false}))
 
 ;; TODO(OmarBasem): Requires design input.
 (defn show-qr-entry
   []
-  (entry {:icon      :i/qr-code
-          :label     (i18n/label :t/show-qr)
-          :on-press  #(js/alert "TODO: to be implemented, requires design input")
-          :danger?   false
-          :sub-label nil
-          :chevron?  false}))
+  (entry {:icon                :i/qr-code
+          :label               (i18n/label :t/show-qr)
+          :on-press            #(js/alert "TODO: to be implemented, requires design input")
+          :danger?             false
+          :accessibility-label :show-qr-code
+          :sub-label           nil
+          :chevron?            false}))
 
 ;; TODO(OmarBasem): to be implemented.
```

Comment:
@OmarBasem, is there an issue for this TODO tracked?
Seems there could easily be a github issue - "get design input for actions in common home actions"

Reply:
same for the on-press.. maybe we can add an issue for these?

Reply:
@J-Son89 There is an issue. Will create an issue now